### PR TITLE
Fix cve 2019 11038

### PR DESCRIPTION
**Pull request by wang0z**
Hi,

I added a test case for CVE-2019-11038 based on the commit from #503 (mejo-).

 Of cause this replaced #506.

### CODE REVIEW

File: tests/xbm/Makemodule.am [hunk 1]
```diff
@@ -5,9 +5,12 @@ libgd_test_programs += \
 	xbm/x10_basic_read
 endif
 
+libgd_test_programs += xbm/github_bug_501
```
Comment (vapier) [inline tests/xbm/Makemodule.am]: move to the top of the file like we do in `gdimagecopy/Makemodule.am`

File: tests/xbm/github_bug_501.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	The pixels of the XBM image are invalid hex which makes the uninitialezed
+	variable be encoded into the output image i.e. information disclosure.
+	The image is 8*2.
+
+	See also <https://github.com/libgd/libgd/issues/501>.
+*/
+
+#include "gd.h"
+#include "gdtest.h"
+#ifdef _WIN32
```
Comment (vapier) [inline tests/xbm/github_bug_501.c]: delete this Windows check

**Comment by wang0z** [comment]
I have made a new commit according your recommendation.

**Comment by vapier** [comment]
thanks, i've merged your commits now (after making a few more style tweaks)

**Pull request closed without merging**